### PR TITLE
[11] Backport web_responsive fix #873 to v11

### PR DESCRIPTION
**Pull request by Ark74**
Adding patch of #873 to v11
Cheers!

### CODE REVIEW

File: web_responsive/static/src/less/main.less [hunk 1]
```diff
@@ -46,6 +46,24 @@ main {
 }
 
 .o_web_client {
+        &.o_fullscreen {
```
Comment (tarteo) [inline web_responsive/static/src/less/main.less]: Can you fit the identation, please?

File: web_responsive/static/src/less/main.less [hunk 2]
```diff
@@ -46,6 +46,24 @@ main {
...
+        #oe_main_menu_navbar {
+            display: none;
+        }
+        .o_main .o_sub_menu {
+            display: none;
+         }
+        .navbar {
+            display: none;
+        }
+        .navbar-default main-nav{
```
Comment (tarteo) [inline web_responsive/static/src/less/main.less]: This doesn't seem right. ```.navbar-default .main-nav```?

File: web_responsive/static/src/less/main.less [hunk 3]
```diff
@@ -46,6 +46,24 @@ main {
...
+            display: none
+        }
+        .main-nav{
```
Comment (tarteo) [inline web_responsive/static/src/less/main.less:53]: Why not use multiple selectors once, like this:
```
#oe_main_menu_navbar,
.o_main .o_sub_menu,
.navbar,
.main-nav {
     display: none;
}
```

**Review by tarteo — CHANGES_REQUESTED** [review]
Thanks!

File: web_responsive/static/src/less/main.less (hunk 1 above)
Comment (Ark74) [reply web_responsive/static/src/less/main.less]: Sure,let me check that.

File: web_responsive/static/src/less/main.less (hunk 2 above)
Comment (Ark74) [reply web_responsive/static/src/less/main.less]: Ohh! You're right.

File: web_responsive/static/src/less/main.less (hunk 3 above)
Comment (Ark74) [reply web_responsive/static/src/less/main.less:53]: Ohh, nice, haven't thought of it.
I'm not too familiar with less, I think of it like css maybe there lays the issue.

Comment (Ark74) [reply web_responsive/static/src/less/main.less:53]: Should `.navbar-default .main-nav` be removed?

Comment (tarteo) [reply web_responsive/static/src/less/main.less:53]: It did nothing before but we can keep it so the intention of the original developer is kept. :P

**Review by tarteo — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]
@Yajo please review and put it on v12

File: web_responsive/static/src/less/main.less (hunk 3 above)
Comment (yajo) [reply web_responsive/static/src/less/main.less:53]: Remove it please. We don't need uneeded code :wink:

**Review by yajo — CHANGES_REQUESTED** [review]

File: web_responsive/static/src/less/main.less (hunk 3 above)
Comment (Ark74) [reply web_responsive/static/src/less/main.less:53]: Done

**Review by yajo — APPROVED** [review]

**Pull request merged**